definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1343,6 +1343,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "09/11/2018",
```

Comment:
I know it doesn't change anything function-wise, but could you please update it to "24/07/2020"?

It's easier to search in these files sometimes without using git blame

Reply:
Mind if I use yesterday's date? Not sure if it counts the day itself and on or off.